common/djangoapps/third_party_auth/provider.py

```diff
@@ -7,7 +7,7 @@
 
 from openedx.core.djangoapps.theming.helpers import get_current_request
 
-from .models import (
+from common.djangoapps.third_party_auth.models import (
```

Comment:
I think this should be harmless, but why was it needed?

Reply:
it was giving error during make docs command.

Reply:
@jmbowman Otherwise this error appears
```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/sphinx/util/pycompat.py", line 89, in execfile_
    exec(code, _globals)
  File "/edx/app/edxapp/edx-platform/docs/guides/conf.py", line 43, in <module>
    django.setup()
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/sap_success_factors/models.py", line 18, in <module>
    from integrated_channels.sap_success_factors.exporters.learner_data import (
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/sap_success_factors/exporters/learner_data.py", line 14, in <module>
    from enterprise.tpa_pipeline import get_user_from_social_auth
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/enterprise/tpa_pipeline.py", line 23, in <module>
    from third_party_auth.provider import Registry
  File "/edx/app/edxapp/edx-platform/common/djangoapps/third_party_auth/provider.py", line 10, in <module>
    from .models import (
  File "/edx/app/edxapp/edx-platform/common/djangoapps/third_party_auth/models.py", line 338, in <module>
    class OAuth2ProviderConfig(ProviderConfig):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/db/models/base.py", line 316, in __new__
    new_class._meta.apps.register_model(new_class._meta.app_label, new_class)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/apps/registry.py", line 227, in register_model
    raise RuntimeError(
RuntimeError: Conflicting 'oauth2providerconfig' models in application 'third_party_auth': <class 'common.djangoapps.third_party_auth.models.OAuth2ProviderConfig'> and <class 'third_party_auth.models.OAuth2ProviderConfig'>.
```